tests/format/misc/plugins/embed-async-parser/with-aync-parser-code-block.md

```diff
@@ -1,5 +1,5 @@
 ```async-parser
```

Comment:
Let's use `markdown` async parser, and remove this test?

https://github.com/prettier/prettier/pull/13129/files